tests/Unit/Domain/TrafficRegulation/RegulationConditionTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Domain\TrafficRegulation;
+
+use App\Domain\TrafficRegulation\RegulationCondition;
+use App\Domain\TrafficRegulation\TrafficRegulation;
+use PHPUnit\Framework\TestCase;
+
+final class RegulationConditionTest extends TestCase
+{
+    public function testGetters(): void
+    {
+        $trafficRegulation = new TrafficRegulation('6598fd41-85cb-42a6-9693-1bc45f4dd392');
```

Comment:
Petit détail, mais ici il ne vaudrait pas mieux utiliser un mock puis un `$this->assertSame(mock, $regulationCondition->getTrafficRegulation()` ? Comme on a fait pour `organization` dans `RegulationOrderRecordTest` par exemple. En effet, on teste déjà `TrafficRegulation->getUuid()` dans les tests unitaires de TrafficRegulation.

Reply:
On pourrait utiliser un mock dans le cas où tu as un objet "complexe". Ici, c'est juste un uuid, du coup, je n'ai pas vu l'intérêt.